### PR TITLE
Fix BBE test case for base-path-and-path

### PR DESCRIPTION
**Pull request by NipunaMarcus**
## Purpose
Fix the syntaxes in base-path-and-path example's test case.

### CODE REVIEW

File: examples/base-path-and-path/tests/base_path_and_path_test.bal [hunk 1]
```diff
@@ -24,11 +24,11 @@ function testFunc() {
     // Send a `GET` request to the specified endpoint.
     var response = httpEndpoint->post("/foo/bar", req);
     match response {
-        http:Response resp => {
+        var resp if resp is http:Response => {
```
Comment (anupama-pathirage) [inline examples/base-path-and-path/tests/base_path_and_path_test.bal]: Shall we remote match and use if check with if (response is http:Response) etc?

Comment (MaryamZi) [reply examples/base-path-and-path/tests/base_path_and_path_test.bal]: Adding sample code:
```ballerina
if (response is http:Response) {
    var jsonRes = check response.getJsonPayload();
    test:assertEquals(jsonRes, payload);
} else {
    test:assertFail(msg = "Failed to call the endpoint:");
}
```

Comment (NipunaMarcus) [reply examples/base-path-and-path/tests/base_path_and_path_test.bal]: will add this change

File: examples/base-path-and-path/tests/base_path_and_path_test.bal [hunk 2]
```diff
@@ -13,23 +13,23 @@ function startService() {
...
     // Invoking the main function.
-    endpoint http:Client httpEndpoint { url: "http://localhost:9090" };
-    // Check whether the server has started. 
+    http:Client httpEndpoint = new("http://localhost:9090");
+    // Check whether the server has started.
     //test:assertTrue(serviceStarted, msg = "Unable to start the service");
     json payload = { "hello": "world" };
     http:Request req = new;
     req.setJsonPayload(payload);
     // Send a `GET` request to the specified endpoint.
```
Comment (VijithaEkanayake) [inline examples/base-path-and-path/tests/base_path_and_path_test.bal]:
```suggestion
    // Send a `POST` request to the specified endpoint.
```

File: examples/base-path-and-path/tests/base_path_and_path_test.bal [hunk 3]
```diff
@@ -13,23 +13,23 @@ function startService() {
...
     var response = httpEndpoint->post("/foo/bar", req);
```
Comment (VijithaEkanayake) [inline examples/base-path-and-path/tests/base_path_and_path_test.bal]:
```suggestion
    var response = httpEndpoint->post("/foo/bar", { "hello": "world" });
```
We don't need create a new request here, we can simply pass the payload to the POST action as an argumemt

**Pull request merged**